elements/lisk-transactions/test/create_signature_object.ts

```diff
@@ -135,4 +135,32 @@ describe('#createSignatureObject', () => {
 			);
 		});
 	});
+
+	describe('when invalid transaction transaction is used', () => {
+		it('should throw an Error if type is not defined', () => {
+			const passphrase = 1;
+			return expect(() =>
+				createSignatureObject({
+					transaction: {} as any,
+					passphrase: (passphrase as unknown) as string,
+					networkIdentifier:
+						multisignatureFixture.testCases.input.networkIdentifier,
+				}),
+			).to.throw('Transaction type is required');
+		});
+	});
+
+	it('should throw an Error if type is invalid', () => {
```

Comment:
I think the state `if type is invalid` can be frames as `if type is not registerd`? because of the type valid but not registered in the sdk? I know it's maybe a silly thing, it just caught my mind.

Reply:
The error thrown by the library under test is `Invalid transaction type.` and there's no registration concept in it. What you suggest happens in the context of the framework and I do agree in framework there should be a test with that wording. I think this should be kept as is in the context of the package.